src/DocScan/Session/Retrieve/Contracts/WatchlistAdvancedCaSearchConfigResponse.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yoti\DocScan\Session\Retrieve\Contracts;
+
+abstract class WatchlistAdvancedCaSearchConfigResponse extends WatchlistSearchConfigResponse
+{
+    /**
+     * @var string $type
+     */
+    private $type;
+
+    /**
+     * @var bool
+     */
+    private $removeDeceased;
+
+    /**
+     * @var bool
+     */
+    private $shareUrl;
+
+    /**
+     * @var CaSourcesResponse
+     */
+    private $sources;
```

Comment:
Not knowing php, I don't immediately see how someone would retrieve _sources_ or _matching_strategy_...?

Reply:
Fixed